numpy/__init__.pyi

```diff
@@ -1,8 +1,91 @@
 # very simple, just enough to start running tests
+#
+import builtins
+from typing import Any, Mapping, List, Optional, Tuple, Union
 
-class ndarray: pass
+from numpy.core._internal import _ctypes
+
+class dtype:
+    alignment: int
+    byteorder: str
+    char: str
+    descr: List[Tuple[str, str]]
```

Comment:
Entries in `descr` can only be three element tuples, with the last item indicating a shape, e.g., 
```
>>> np.dtype([('x', 'f4'), ('y', np.float32), ('z', 'f4', (2,2))]).descr
[('x', '<f4'), ('y', '<f4'), ('z', '<f4', (2, 2))]
```